src/lib/isMobilePhone.js

```diff
@@ -82,7 +82,8 @@ const phones = {
   'fr-MQ': /^(\+?596|0|00596)[67]\d{8}$/,
   'fr-RE': /^(\+?262|0|00262)[67]\d{8}$/,
   'he-IL': /^(\+972|0)([23489]|5[012345689]|77)[1-9]\d{6}$/,
-  'hu-HU': /^(\+?36|06)(20|30|31|50|70)\d{7}$/,
+  'hr-HR': /^(\+?385|0)9(1|2|5|7|8|9)\d{7}$/,
+  'hu-HU': /^(\+?36)(20|30|70)\d{7}$/,
```

Comment:
@atom19-i looks like you fixed the conflicts in `README.md` but introduced an error in `isMobilePhone.js`. Just put back the existing version of `hu-HU`and everything should be okay.

```suggestion
  'hr-HR': /^(\+?385|0)9(1|2|5|7|8|9)\d{7}$/,
  'hu-HU': /^(\+?36|06)(20|30|31|50|70)\d{7}$/,
```